python/tests/utils/latex_test.py

```diff
@@ -19,9 +19,12 @@
 
 test_cases_latex = (
```

Comment:
Would you mind adding a comment about what this is meant to test? Is it supposed to be normalizing LaTeX to a more canonical LaTeX, or something else?

Reply:
When converting from our markup into LaTeX, pylatexenc turns `"` into `{''}`. The tested function applies heuristics to turn these into appropriate opening/closing quotes with the braces removed. I tried to clarify this now in the code.

To be fair, this logic has been part of the legacy library and I just ported it without giving it too much thought.